third-party/realdds/include/realdds/dds-stream-profile.h

```diff
@@ -32,9 +32,11 @@ union dds_stream_uid
     }
 
     dds_stream_uid( int sid_, int index_ )
-        : sid( static_cast< int16_t >( sid_ ))
-        , index( static_cast< int8_t >( index_ ))
-    {}
+    {
+        whole = 0;  // it covers an extra byte, which needs to be 0

```

Comment:
No need to change in default constructor as well?

Reply:
No, the default uses the `= 0` from above.